src/views/portal/AcceptedMessage.tsx

```diff
@@ -0,0 +1,139 @@
+				<strong>visible at all times</strong>.
+			</p>
+
+			<h4>Venue Information</h4>
+			<p>
+				Hack at UCI 2023 will take place (not overnight) on Friday evening,
+				all-day Saturday, and approximately the first half of Sunday on-campus
+				at UC Irvine Student Center.{" "}
+				<strong className={styles.red_text}>
+					You must check in on all three event days in order to be eligible to
+					win prizes, specifically attending the opening ceremony on Friday,
+					project expo for judging on Sunday, and completing the Midway check-in
+					form on Saturday
+				</strong>
+				. Hackathon events will take place in the following rooms:
+			</p>
+			<p>
+				<strong>Pacific Ballroom Lobby:</strong> Check In
+				<br />
+				<strong>Pacific Ballroom ABCD:</strong> Hacking Space
+				<br />
+				<strong>Woods Cove B + C:</strong> Hacking Space
+				<br />
+				<strong>Moss Cove A + B:</strong> Workshops and Socials
+			</p>
+			<p>
+				Map:{" "}
+				<a href="https://map.uci.edu/?id=463#!m/255880">
+					https://map.uci.edu/?id=463#!m/255880
+				</a>
+			</p>
+			<p>
+				Floor Map:{" "}
+				<a href="https://conference.studentaffairs.uci.edu/node/24">
+					https://conference.studentaffairs.uci.edu/node/24
+				</a>
+			</p>
+			<h4>On-Campus Parking</h4>
+			<p>
+				<strong>Attendees will be responsible for their own parking.</strong>{" "}
+				The closest parking structure to our event is the UCI Student Center
+				Parking Structure. This structure provides both pay-by-space ($2 per
+				hour) and permit dispensers ($1 per hour, $13 for a day). You may
+				purchase a permit at any information kiosk, dispenser, or through
+				UCI&#39;s ParkbyPlate signage located in select parking areas. All
+				options accept major credit cards (Visa, MasterCard, American Express,
+				Discover). These dispensers are available from 7AM-7PM.
+			</p>
+			<p>
+				To enter the venue from the parking structure, cross Pereira Dr. and
+				enter the building with the large arch labeled “Student Center”. Once
+				you enter the building, walk straight down the hallway till the stairs.
+				Continue down the steps (there&#39;s a ramp to the left for
+				accessibility) and check in at the large desk in the lobby. If you
+				cannot get inside the venue at any point, please let an organizer know
+				via Slack, and we&#39;d be happy to help.
+			</p>
+			<p>
+				Map:{" "}
+				<a href="https://map.concept3d.com/?id=463#!ce/8403?ct/55161,27429,35849?m/367524?s/">
+					https://map.concept3d.com/?id=463#!ce/8403?ct/55161,27429,35849?m/367524?s/
+				</a>
+			</p>
+
+			<h4>Pre-Hack-At-UCI Checklist</h4>
+			<ul>
+				<li>Confirm your attendance by clicking on the button below</li>
+				<li>Sign the liability waiver once it has been sent to your email</li>
+				<li>Join the Slack once the invite has been sent to your email</li>
+				<li>
+					Bring a <strong>photo ID for check</strong> in on Day 1, and your{" "}
```

Comment:
Question: should "check-in" also be rendered as important? Also, since check-in is used as a noun here, it should be hyphenated.